rpcserver.go

```diff
@@ -2812,6 +2812,11 @@ func (r *rpcServer) arbitratorPopulateForceCloseResp(chanPoint *wire.OutPoint,
 
 			forceClose.PendingHtlcs = append(forceClose.PendingHtlcs, htlc)
 
+		case contractcourt.ReportOutputAnchor:
+			if report.LimboBalance == 0 {
+				forceClose.AnchorRecovered = true
```

Comment:
should have a comment

Reply:
comment added. Also changed condition to `RecoveredBalance != 0`, because an anchor may also be lost.

Reply:
shouldn't we attempt to report all the states it could be in?

Reply:
all states exposed